### PR TITLE
VCB-19 Fix ALIGN is not recognized

### PR DESCRIPTION
**Pull request by PhilSchni**
Fixed issue that ALIGN was not correctly recognized.

### CODE REVIEW

File: test/assembler/parser.test.ts [hunk 1]
```diff
@@ -34,18 +36,19 @@ describe('parse code', function () {
...
     expect(ast.areas[1].instructions[0].line).toBe(8)
     expect(ast.areas[1].instructions[1].name).toBe('MOVS')
     expect(ast.areas[1].instructions[1].label).toBe('label1')
     expect(ast.areas[1].instructions[1].options).toEqual(['R2', '#456'])
     expect(ast.areas[1].instructions[1].line).toBe(9)
-    expect(ast.areas[1].instructions[2].name).toBe('MOVS')
-    expect(ast.areas[1].instructions[2].label).toBe('label2')
-    expect(ast.areas[1].instructions[2].options).toEqual(['R3', '#789'])
-    expect(ast.areas[1].instructions[2].line).toBe(12)
+    expect(ast.areas[1].instructions[2].name).toBe('ALIGN')
```
Comment (m-dot-wigger) [inline test/assembler/parser.test.ts:47]: check that there are no options provided (length zero)

File: src/components/editor/plugins/assembly/assembly.grammar [hunk 2]
```diff
@@ -87,6 +88,18 @@ BranchInstructionWithoutLabel {
   RegisterLiteral
 }
 
+DataInstructionDeclaration {
+    Label (kw<"DCD"> | kw<"DCW"> | kw<"DCB">) IntegerLiteral
```
Comment (m-dot-wigger) [inline src/components/editor/plugins/assembly/assembly.grammar:92]: Add possiblity for String Literal

**Review by m-dot-wigger — CHANGES_REQUESTED** [review]

File: src/components/editor/plugins/assembly/assembly.grammar (hunk 2 above)
Comment (m-dot-wigger) [reply src/components/editor/plugins/assembly/assembly.grammar:92]: as discussed closed

Comment (PhilSchni) [reply src/components/editor/plugins/assembly/assembly.grammar:92]: As discussed with Leo and Michael, we will not implement this.

File: test/assembler/parser.test.ts (hunk 1 above)
Comment (PhilSchni) [reply test/assembler/parser.test.ts:47]: Changed this.

**Review by m-dot-wigger — APPROVED** [review]

**Pull request merged**